src/stores/businessStore.ts

```diff
@@ -186,6 +191,11 @@ export const useBusinessStore = defineStore('business', {
       return (this.getBusinessState === EntityState.HISTORICAL)
     },
 
+    /** Is True if entity is a Sole Proprietorship or General Partnership. */
+    isTypeFirm (): boolean {
+      return (this.isTypeSoleProp || this.isTypePartnership)
```

Comment:
Are these 2 getters declared anywhere in this app? I can't find them.

Reply:
Removing this one.

Just found out there's already a similar one:

![image](https://github.com/bcgov/business-filings-ui/assets/17242795/fa175840-ebd1-4585-afa4-2581dced09b5)

Reply:
This may also be why the unit tests failed (with getLegalName returning undefined).

PS You can (and should) run both linting and the unit tests locally:
```
npm run lint
npm run test:unit
```